tests/test_run.py

```diff
@@ -1,16 +1,20 @@
 import numpy as np
 import pytest
 
-from stable_baselines3 import A2C, PPO, SAC, TD3, DQN
+from stable_baselines3 import A2C, PPO, SAC, TD3, DQN, DDPG
 from stable_baselines3.common.noise import NormalActionNoise, OrnsteinUhlenbeckActionNoise
 
 normal_action_noise = NormalActionNoise(np.zeros(1), 0.1 * np.ones(1))
 
 
+@pytest.mark.parametrize('model_class', [TD3, DDPG])
 @pytest.mark.parametrize('action_noise', [normal_action_noise, OrnsteinUhlenbeckActionNoise(np.zeros(1), 0.1 * np.ones(1))])
-def test_td3(action_noise):
-    model = TD3('MlpPolicy', 'Pendulum-v0', policy_kwargs=dict(net_arch=[64, 64]),
-                learning_starts=100, verbose=1, create_eval_env=True, action_noise=action_noise)
+def test_dpg(model_class, action_noise):
```

Comment:
`dpg` looks like a typo for `ddpg` but I think it's probably intended since it's both TD3 and DDPG. Can we disambiguate this? e.g. `test_deterministic_pg`?

Reply:
good point, yes, not a typo, I will change that
```suggestion
def test_deterministic_pg(model_class, action_noise):
```